SimpleRabbit.NetCore/Service/BasicRabbitService.cs

```diff
@@ -50,6 +50,7 @@ protected ConnectionFactory Factory
                     NetworkRecoveryInterval = TimeSpan.FromSeconds(_config.NetworkRecoveryIntervalInSeconds ?? DefaultNetworkRecoveryInterval),
                     TopologyRecoveryEnabled = _config.TopologyRecoveryEnabled ?? true,
                     RequestedHeartbeat = TimeSpan.FromSeconds(_config.RequestedHeartBeat ?? DefaultRequestedHeartBeat),
+                    DispatchConsumersAsync = _config.UseAsyncDispatch ?? true,
                 };

```

Comment:
If we are using this flag, we need to attach different event based on this flag

Reply:
This flag is a backward compatible layer between new behavior and previous behavior. People should set it to false when synchronised event handler is used.
We are using QueueService, and with the latest version, it is async, so we should be fine. @ashikgrg

Reply:
If this flag is true, we should use `EventingBasicConsumer` instead of `AsyncEventingBasicConsumer` otherwise it throws.